scripts/EN/Harry

```diff
@@ -5690,18 +5690,18 @@ Area(0xca); // 0x72d8: 00001026
 CleanScreen(); // 0x72de: 00000064
 LoadBackground(0xaf); // 0x72e2: 00001069
 DrawBackground(); // 0x72e8: 00000065
-ShowText((1596)"At that time I remembered. I searched my pockets...got it. The pet bottle. The vermilion liquid is inside it. I scooped it up in the hospital."); // 0x72ec: 0000100c
+ShowText((1596)"At that time I remembered.¥nW2I searched my pockets…W2got it.¥nW2The plastic bottle.¥nW2The vermilion liquid is inside it.¥nW2I scooped it up in the hospital."); // 0x72ec: 0000100c
 HandleInput(); // 0x72f2: 00000010
-ShowText((1597)"The red of Cybil's eyes looked a lot like the red of that liquid. I hold up the pet bottle so that Cybil can see it and shake it left and right."); // 0x72f6: 0000100c
+ShowText((1597)"The red of Cybil's eyes looked a lot like the red of that liquid.¥nW2I hold up the bottle W1so that Cybil can see it W1and shake it left and right."); // 0x72f6: 0000100c
```

Comment:
Here I just removed plastic because it felt redundant at this point.

Feel free to say "bottle" or "plastic bottle" as it works best in your language.